test/integration/dredd-test.coffee

```diff
@@ -562,17 +562,26 @@ describe 'Dredd class Integration', ->
       )
     )
 
-    it('recognizes all 3 transactions', ->
-      assert.equal(matches.length, 3)
+    it('recognizes all 6 transactions', ->
+      assert.equal(matches.length, 6)
     )
     it('the transaction #1 is skipped', ->
       assert.equal(matches[0][1], 'skip')
     )
     it('the transaction #2 is skipped', ->
       assert.equal(matches[1][1], 'skip')
     )
-    it('the transaction #3 is not skipped (status 200)', ->
-      assert.notEqual(matches[2][1], 'skip')
+    it('the transaction #3 is skipped', ->
+      assert.equal(matches[2][1], 'skip')
+    )
+    it('the transaction #4 is skipped', ->
+      assert.equal(matches[3][1], 'skip')
+    )
+    it('the transaction #5 is not skipped (status 200)', ->
+      assert.notEqual(matches[4][1], 'skip')
+    )
+    it('the transaction #6 is not skipped (status 200)', ->
+      assert.notEqual(matches[5][1], 'skip')
```

Comment:
Good. But given the changes, I would turn this test into sth like

- when Swagger document has multiple responses
  - the responses with status code 200 are not skipped
  - the responses with status code 500 are skipped

I'd keep `recognizes all 6 transactions`, but the rest would iterate over expected values and inspect the matches more. I think this way we rely too much on what is in the fixture and what the structure is and the outcome of the test isn't very clear anymore.